packages/web-app-files/src/components/PublicLinksSidebar/EditPublicLink.vue

```diff
@@ -22,57 +21,49 @@
           @roleSelected="$_selectRole"
         />
       </oc-grid>
-      <div class="oc-mb uk-grid-small uk-flex uk-flex-middle" uk-grid>
+      <div class="oc-mb uk-grid-small uk-flex" uk-grid>
         <div v-if="$_expirationDate" class="uk-width-1-1 uk-width-2-5@m">
-          <label class="oc-label" for="oc-files-file-link-expire-date">
-            <span v-translate>Expiration date:</span>
-            <translate v-if="$_expirationDate.enforced" tag="em">(required)</translate>
-          </label>
           <div class="uk-position-relative">
             <oc-datepicker
               id="oc-files-file-link-expire-date"
               :key="'oc-datepicker-' + expireDate"
-              :class="{ 'uk-form-danger': !$_expirationIsValid }"
+              :label="
+                `${$gettext('Expiration date')} ${
```

Comment:
Could you create a computed property for the expiration date and have two different $gettext strings for it? in case it's required `$gettext('Expiration date (required)')` and in case it's not required `$gettext('Expiration date')`? I would like to avoid concatenating translated partials. That might break in other languages.